tests/test_instance.py

```diff
@@ -428,3 +428,23 @@ def test_create_channel_not_supported():
     # The exception was raised with a descriptive error
     assert "instances/hello-world-32" in str(exc)
     assert "grpc" in str(exc)
+
+
+def test_str():
+    instance_str = str(
+        pypim.Instance(
+            name="instances/hello-world-32",
+            definition_name="definitions/my-def",
+            ready=False,
+            status_message="Loading.",
+            services={
+                "http": "http://example.com",
```

Comment:
suggestion: rename the service to "my-http", to be sure that the assert at line 449 doesn't capture the word `http` from the url.